### PR TITLE
Bump Google.Cloud.Diagnostics.* to GA

### PR DESCRIPTION
**Pull request by iantalarico**
@jskeet mind doing a quick double check?

### CODE REVIEW

**Comment by jskeet** [comment]
Will do when I get a chance, but I don't know when I'll be back on a laptop

**Comment by iantalarico** [comment]
@jskeet No rush, I just wanted a second set of eyes.  I keep forgetting your on vacation.  I'll add Mike.

File: apis/apis.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
       "Microsoft.AspNet.WebApi.Core": "5.2.3",
```
Comment (jskeet) [inline apis/apis.json]: Is it worth checking these dependency versions are appropriate? Are there more recent versions that are worth picking up, for example?

Comment (iantalarico) [reply apis/apis.json]: Good catch, bumped all to the latest stable release, thanks Jon!

**Comment by iantalarico** [comment]
Thanks Jon :)

**Pull request merged**